src/olympia/reviewers/views.py

```diff
@@ -909,6 +909,14 @@ def review(request, addon, channel=None):
     else:
         flags = []
 
+    try:
+        whiteboard = Whiteboard.objects.get(pk=addon.pk)
+    except Whiteboard.DoesNotExist:
+        whiteboard = Whiteboard(pk=addon.pk)
```

Comment:
shouldn't use this `get_or_create` too?

Reply:
same in these other places - it's not being saved until it's needed.  (I've not thought through the logic about whether that's going to work)

Reply:
I'm not sure about edge cases, but I did test this on my dev instance. I thought it didn't make sense to create during viewing, if that breaks anything please let me know :)

Reply:
Does `ModelForm` need an instance anyway?  Afaik in other places we don't create objects - the form creates them on save (you'll need to pass in the add-on ID, obviously.

Reply:
I'd need to pass in the add-on id, and previous content. This comes close to providing an instance. If you can tell me how other code does this I'm happy to make changes, but for now I think we should just go ahead and push :)r